docs/using_stips/psf_grid.rst

```diff
@@ -2,97 +2,19 @@ The STIPS PSF Grid
 ==================
 .. note::
 
-    The STIPS PSF Grid creation and configuration process is still in flux. At 
-    the moment there is a somewhat confusing conflation of PSF convolution size
-    (which should be mostly to do with available memory and computation speed)
-    and how many individual PSF interpolations are used from a given grid. In
-    the future, these considerations will be split apart, so the documentation
-    here is subject to change.
+    The STIPS PSF Grid creation and configuration process is still in flux. At the moment, there is a somewhat confusing conflation of PSF convolution size (which should be mostly to do with available memory and computation speed) and how many individual PSF interpolations are used from a given grid. In the future, these considerations will be split apart, so the documentation here is subject to change.
 
 Overview
 --------
 
-Starting with STIPS 1.0.8, STIPS supports (and requires) the use of webbpsf PSF 
-grids when creating PSFs for convolution. When the STIPS image convolution step
-is performed, it will divide up the image into squares (with size set by the
-PSF convolution size configuration item), and a PSF will be generated for each
-square, with the PSF detector location set to the centre of that square. As a
-result, different regions of the detector will have different PSFs applied to
-them.
+Starting with STIPS 1.0.8, STIPS supports (and requires) the use of webbpsf PSF grids when creating PSFs for convolution. When the STIPS image convolution step is performed, it will divide up the image into squares (with size set by the PSF convolution size configuration item), and a PSF will be generated for each square, with the PSF detector location set to the centre of that square. As a result, different regions of the detector will have different PSFs applied to them.
```

Comment:
```suggestion
Starting with STIPS 1.0.8, STIPS supports (and requires) the use of WebbPSF PSF grids when creating PSFs for convolution. When the STIPS image convolution step is performed, it will divide up the image into squares (with size set by the PSF convolution size configuration item), and a PSF will be generated for each square, with the PSF detector location set to the centre of that square. As a result, different regions of the detector will have different PSFs applied to them.
```